tekton/release-pipeline.yaml

```diff
@@ -8,53 +8,149 @@ spec:
     - name: package
       description: package to release
       default: github.com/tektoncd/dashboard
+    - name: gitRevision
+      description: the git revision to release
     - name: imageRegistry
-      default: gcr.io/tekton-releases
+      description: The target image registry
+      default: gcr.io
+    - name: imageRegistryPath
+      description: The path (project) in the image registry
+      default: tekton-releases
     - name: versionTag
       description: The X.Y.Z version that the artifacts should be tagged with
+    - name: releaseBucket
+      description: >-
+                      bucket where the release is stored. The bucket must be
+                      project specific.
+      default: gs://tekton-releases-nightly/dashboard
     - name: releaseAsLatest
-      description: Whether to tag and publish this release as the latest
+      description: Whether to tag and publish release as Dashboard's latest
       default: "true"
-  resources:
-    - name: dashboard-source-repo
-      type: git
-    - name: bucket-for-dashboard
-      type: storage
-    - name: builtDashboardImage
-      type: image
+    - name: platforms
+      description: Platforms to  build images for (linux/amd64,linux/arm64 ...)
+      default: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
+    - name: serviceAccountPath
+      description: >-
+                      The path to the serviceaccount file within
+                      the release-secret workspace
+  workspaces:
+    - name: workarea
+      description: The workspace where the repo will be cloned.
+    - name: release-secret
+      description: >-
+                      The secret that contains a service account
+                      authorized to push to the imageRegistry and
+                      to the output bucket
+  results:
+    - name: commit-sha
+      description: the sha of the commit that was released
+      value: $(tasks.git-clone.results.commit)
+    - name: release
+      description: the URL of the release file
+      value: $(tasks.report-bucket.results.release)
+    - name: release-ro
+      description: the URL of the readonly release file
+      value: $(tasks.report-bucket.results.release-ro)
+    - name: openshift-release
+      description: the URL of the OpenShift release file
+      value: $(tasks.report-bucket.results.openshift-release)
+    - name: openshift-release-ro
+      description: the URL of the OpenShift readonly release file
+      value: $(tasks.report-bucket.results.openshift-release-ro)
   tasks:
+    - name: git-clone
+      taskRef:
+        name: git-clone
```

Comment:
should reference the catalog in the full setup instructions to include details of the additional tasks that are required when running on a cluster other than dogfooding